src/leiningen/new/cryogen/themes/blue/html/base.html

```diff
@@ -95,5 +95,7 @@
 <script src="//maxcdn.bootstrapcdn.com/bootstrap/3.3.0/js/bootstrap.min.js"></script>
 {% script "js/highlight.pack.js" %}
 <script>hljs.initHighlightingOnLoad();</script>
+{% if post.klipse %} {{post.klipse|safe}} {% endif %}
+{% if page.klipse %} {{page.klipse|safe}} {% endif %}
```

Comment:
Using the blue theme I can't spot any difference, but it makes sense that there would be.

If it's totally disabled, then any code blocks not marked as evalable would have no highlighting. Probably not the most common use case, but still.

 Another way could be to use enlive to tag any blocks marked for eval with the class `nohilight`. What do you think?

(We can just count any strings with a leading dot that are values in the `:settings` map as evalable class names to make it future proof when new selectors are added.)